{{ cookiecutter.repo_name }}/.govcookiecutter/department_frameworks/README.md

```diff
@@ -0,0 +1,30 @@
+# `department_frameworks` folder
```

Comment:
Is it clear what type of framework? For some reason I thought front-end framework, which is something most departments have, for instance.

Reply:
Probably failing YAGNI here, but thought I'd make the folder generic enough so that **if** there are other frameworks for a given department (not just AQA) these could be included in future. Don't have an example really to hand - happy to change this back @maxf